### PR TITLE
Fix 8706. Mountain tool clipping issues after action refactor

### PR DESCRIPTION
**Pull request by duncanspumpkin**
Ultimately the mountain tool needs further refactoring when ported to a game action. For now this will fix the issue.

### CODE REVIEW

File: src/openrct2/world/Map.cpp [hunk 1]
```diff
@@ -2393,8 +2402,8 @@ void game_command_place_large_scenery(
 
         if (!gCheatsDisableClearanceChecks
             && !map_can_construct_with_clear_at(
-                   curTile.x, curTile.y, zLow, zHigh, &map_place_scenery_clear_func, bl, flags, &supportsCost,
-                   CREATE_CROSSING_MODE_NONE))
+                curTile.x, curTile.y, zLow, zHigh, &map_place_scenery_clear_func, bl, flags, &supportsCost,
+                CREATE_CROSSING_MODE_NONE))
```
Comment (Broxzier) [inline src/openrct2/world/Map.cpp]: Clang-format wants the indention here restored.

If your clang-format formatted it this way... I noticed a similar issue earlier. The version on Travis probably doesn't match your local machine.

Comment (duncanspumpkin) [reply src/openrct2/world/Map.cpp]: Yes it's quite annoying. I seem to be hitting a few clang format bugs.

**Comment by duncanspumpkin** [comment]
One thing i found in testing is it actually meant you could change surfaces that you don't own!

File: src/openrct2/world/Map.cpp (hunk 1 above)
Comment (IntelOrca) [reply src/openrct2/world/Map.cpp]: Might be better to make it a new nested if block.

**Comment by ZehMatt** [comment]
I've been lately thinking that its actually safe to just use GameActions::Execute if we always pass down the flags. Given that when its executed in MP it will carry the network flag otherwise requires a round trip, so top level commands should invoke nested ones with the flags always set.

**Pull request closed without merging**